src/server/pfs/server/url.go

```diff
@@ -211,23 +211,24 @@ func (d *driver) getFileURL(ctx context.Context, taskService task.Service, URL s
 			}
 			return nil
 		}
-		var count int64
+		var numObjects, size int64
 		if err := src.Iterate(ctx, func(fi *pfs.FileInfo, file fileset.File) error {
 			if fi.FileType != pfs.FileType_FILE {
 				return nil
 			}
+			numObjects++
```

Comment:
I think the update should be after the check. Path ranges are inclusive / exclusive.